packages/mobile/src/home/HomeTokenBalance.tsx

```diff
@@ -0,0 +1,146 @@
+import Colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+import BigNumber from 'bignumber.js'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Image, StyleSheet, Text, TouchableOpacity, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import { HomeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Dialog from 'src/components/Dialog'
+import { Namespaces } from 'src/i18n'
+import InfoIcon from 'src/icons/InfoIcon'
+import ProgressArrow from 'src/icons/ProgressArrow'
+import { getLocalCurrencySymbol } from 'src/localCurrency/selectors'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { tokensWithBalanceSelector, totalTokenBalanceSelector } from 'src/tokens/selectors'
+
+function TokenBalance() {
+  const tokenBalances = useSelector(tokensWithBalanceSelector)
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+  const totalBalance = useSelector(totalTokenBalanceSelector)
+  if (tokenBalances.length == 0) {
+    return (
+      <Text style={styles.balance}>
+        {localCurrencySymbol}
+        {'0.00'}
+      </Text>
+    )
+  } else if (tokenBalances.length == 1) {
```

Comment:
I know this probably works fine for this case but lets use `===` for equals checks :) The `==` is prone to false positives in many cases. Foe example, if `tokenBalances.length` returns `undefined`, `tokenBalances.length == 0` will still equal `true`.

It's just generally a good practice to use the strict equality operator `===`